src/platforms/hosted/jlink.c

```diff
@@ -317,6 +325,24 @@ static bool jlink_get_version(void)
 	return true;
 }
 
+static bool jlink_get_extended_capabilities(void)
+{
+	uint8_t buffer[32U];
+
+	if (!jlink_simple_query(JLINK_CMD_INFO_GET_PROBE_EXTENDED_CAPABILITIES, buffer, sizeof(buffer)))
+		return false;
+
+	uint32_t ext_caps[4];
+	ext_caps[0] = read_le4(buffer, 0);
+	ext_caps[1] = read_le4(buffer, 4);
+	ext_caps[2] = read_le4(buffer, 8);
+	ext_caps[3] = read_le4(buffer, 12);
```

Comment:
```suggestion
    for (size_t i = 0; i < 4; i++)
		ext_caps[i] = read_le4(buffer, i * 4U);
```

Reply:
Applied and changed the debug printout to use this loop as well.